examples/common/pigweed/rpc_config.gni

```
@@ -31,7 +31,7 @@ rpc_sources = [
 ]
 
 rpc_deps = [
-  "$dir_pw_hdlc:pw_rpc",
+  "$dir_pw_hdlc:default_addresses", 
```

restyle is complaining about the extra space at the end of the line
```suggestion
  "$dir_pw_hdlc:default_addresses",
```